test/integration/highlight_editor_spec.mjs

```diff
@@ -2254,7 +2260,7 @@ describe("Highlight Editor", () => {
           await page.waitForSelector(`${editorSelector} button.delete`);
           await page.click(`${editorSelector} button.delete`);
           await waitForSerialized(page, 0);
-          await page.waitForSelector("#editorUndoBar:not([hidden])");
+          await page.waitForSelector("#editorUndoBar", { visible: true });
 
           await page.waitForSelector("#editorUndoBarCloseButton");
```

Comment:
Here, and probably elsewhere below, do we not also (at least for consistency) need to wait for this button to be *visible* before clicking on it?

Reply:
Good find! Yes, that'd be a good idea indeed. I have updated the patch to do that now (this was the only usage of that button).